src/pages/Product/Product.js

```diff
@@ -0,0 +1,362 @@
+import React, { useState, useEffect } from 'react';
+import { withRouter } from 'react-router-dom';
+import styled, { css } from 'styled-components';
+import { flexSet } from '../../styles/Variable';
+
+function Product({ theme }) {
+  const [inputValue, setInputValue] = useState('');
+  const [commentList, setCommentList] = useState([]);
+  const [opacity, setOpacity] = useState(true);
+
+  useEffect(() => {
+    fetch('/data/comment.json')
+      .then(res => res.json())
+      .then(res => setCommentList(res.data.comment));
+  }, []);
+
+  const onChangeCommentInput = e => {
+    setInputValue(e.target.value);
+    e.target.value.length > 0 ? setOpacity(false) : setOpacity(true);
```

Comment:
빈 스트링은 falsy값이니까
 ```suggestion
setOpacity(!e.target.value)
```
로 표현 가능해보입니다.

Reply:
오... 감사합니다.